Automattic/wc-calypso-bridge
# Remove blocks default / Mitigate object cache issue

### Changes proposed in this Pull Request:



Closes #1342 , closes #1367 .

- #1342
- #1367


### Remove defaulting to blocks

As of WC 8.3.0, cart/checkout blocks are the default. With this PR we remove our code that was defaulting to blocks

### Mitigate Object Cache Issue

Code has been deployed to all server pools that mitigate the issues with Object Cache. Removing `wp_cache_flush` and `usleep` as not needed anymore




### How to test the changes in this Pull Request:




1. Push this PR on your WoA dev site
2. Comment out 
```php
			// Set the operation as completed if the store is active for more than 60 minutes.
			if ( WCAdminHelper::is_wc_admin_active_for( 60 * MINUTE_IN_SECONDS ) ) {
				update_option( $this->option_prefix . $operation, 'completed', 'no' );
				$this->write_to_log( $operation, 'completed (60 minutes)' );

				return;
			}
```

3. Delete all pages from the site and empty trash
4. SSH into the site and run `wp option delete wc_calypso_bridge_one_time_operation_woocommerce_create_pages`
5. Visit `/wp-admin` and go to pages. You'll see 5 new Woo related pages



### Other information:




### FOR PR REVIEWER ONLY:
